code/style.css

```diff
@@ -1,7 +1,247 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+body {
+  font-family: 'Open Sans Condensed', sans-serif;
+  margin: 0;
+  width: 100%;
+}
 
-.active + .answer {
-  /* some styling here */
+header {
+  position: relative;
+  top: 0;
+  left: 0;
+  display: flex;
+  z-index: 1000;
+  background-image: url('./Moutain\ movie\ compressed.mp4');
+  width: 100%; 
+  height: 100vh;
+  object-fit: cover;
+  justify-content: center;
+  align-items: center;
+  overflow: hidden;
 }
+
+.text {
+  position: absolute;
+  color: #ffffff;
+  line-height: 1.5;
+  text-align: center;
+}
+
+.menu-button {
+  background: none;
+  border: solid #ffffff;
+  color: #ffffff;
+  border-radius: 40px;
+  font-size: 15px;
+  padding: 15px;
+  text-decoration: none;
+  box-sizing: border-box;
+  cursor: pointer;
+}
+
+.menu-button:hover {
+  background: #ffffff; 
+  color: #333333;
+  font-weight: bold;
+}
+
+.text h1 {
+  font-size: 60px;
+}
+
+.text p {
+  font-size: 16px;
+  margin-bottom: 20px; 
+  padding: 10px;
+}
+
+fieldset {
+  display: block;
+  margin-inline-start: 1px;
+  margin-inline-end: 1px;
+  margin-top: 5em;
+}
+
+.grid-container {
+   display: grid;
+   grid-template-columns: repeat (2, 1fr);
+   grid-template-rows: repeat (3, 1fr);
+   grid-column-gap: 0px;
+   grid-row-gap: 0px;
+   padding: 20px; 
+  }
+
+  input[type=text], [type=email], select {
+    width: 100%;
+    padding: 12px 20px;
+    margin: 8px 0;
+    display: inline-block;
+    border: 1px solid rgb(73, 167, 120);
+    border-radius: 4px;
+    box-sizing: border-box;
+  }
+
+  .form-button {
+    background: none;
+    border: 1px solid rgb(73, 167, 120);
+    color: rgb(73, 167, 120);
+    border-radius: 20px;
+    font-size: 15px;
+    padding: 5px;
+    text-decoration: none;
+    box-sizing: border-box;
+    cursor: pointer;
+    width: 70px;
+  }
+
+  .form-button:hover {
+    background:rgb(73, 167, 120);
+    color: #ffffff;
+    font-weight: bold;
+  }
+
+  .signup { 
+    grid-area: 1 / 1 / 4 / 5;
+  }
+
+  .signup fieldset {
+    padding: 30px 60px;
+  }
+
+  legend {
+    color: rgb(73, 167, 120);
+  }
+
+  .accordion { 
+    grid-area: 4 / 1 / 4 / 5;
+  }
+
+  .accordion__button {
+    display: block;
+    width: 100%;
+    padding: 15px;
+    border: none;
+    outline: none;
+    cursor: pointer;
+    background: #555454;
+    color: #ffffff;
+    text-align: left;
+    transition: background 0.2s;
+    font-size: 17px;
+  }
+
+  .accordion__button::after {
```

Comment:
Love the smooth animation on the accordion and that you managed to change icon when expanded! Maybe you can show me how? Not sure I follow the code.